LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Shouldn't we add a header in every source-code of the project?

Reply:
Something like:
```
/**
 * Copyright (c) Hathor Labs and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Reply:
We already have this header in all files, only the `LICENSE` file in the root dir that was missing.